pkg/state/filter_test.go

```diff
@@ -14,23 +14,172 @@ var _ = Describe("FilterOut", func() {
 		interfacesFilterGlob glob.Glob
 	)
 
+	Context("when there is a linux bridge with gc-timer and hello-timer", func() {
```

Comment:
States you have used there have way more stuff in them than these two attributes. That makes it terrifying to read through. Any reason to keep all the options there?

Reply:
Just copy paste from real linux-bridge, I will clean that up, at the end I just one to keep some to be sure that only the specific ones are deleted.